packages/eslint-plugin/src/configs/react.js

```diff
@@ -343,8 +343,20 @@ const getOverrides = () => [
       'import/no-webpack-loader-syntax': 'off', // this is ok in docs
     },
   },
+  {
+    files: [...configHelpers.devDependenciesFiles],
+    rules: {
+      'import/no-extraneous-dependencies': [
+        'error',
+        {
+          packageDir: [process.cwd(), configHelpers.findGitRoot()],
+        },
+      ],
+    },
+  },
 ];
 
+// @ts-ignore
```

Comment:
**q:** why the sudden need of turning off type checker ?

Reply:
can you pls create issue for that and add it to this pragma so it's properly tracked ?

Reply:
I'm not sure that's worth creating an issue about, either in our repo or typescript. It's not likely that we'll change the way `overrides` is defined here unless eslint starts supporting config functions (see comment by definition of `getOverrides`). I also don't necessarily think the TS error here is incorrect--you could maybe make an argument that it's incorrect since `overrides` is optional, but in any case I don't find it worth the effort to pursue since it's an uncommon case for us and isn't actually causing any serious problems.

Reply:
I don't know what are the best practices in this repo overall, this approach is what worked for most of my carer in long run. 

can you at least add comment why is this turn checker off needed ? so anyone running into this code will immediately understand ? thx

Reply:
I agree in general, but in some cases such as this where it's an extremely uncommon issue (I've only encountered this one instance of it in our entire repo) which isn't causing any meaningful problems, and it's not even clear that typescript's behavior is wrong, I just don't think it's important enough to justify the nontrivial investment of time in writing up as a typescript issue. And I'm not going to create noise in our own issue queue about something we have no intent to change.